ocaml/libs/tracing/tracing.ml

```diff
@@ -620,6 +625,15 @@ module Export = struct
           span_list
       with e -> debug "Tracing: ERROR %s" (Printexc.to_string e)
 
+    let flush_spans () =
+      let span_list = Spans.since () in
+      get_tracer_providers ()
+      |> List.iter (fun x ->
+             if x.TracerProvider.enabled then
```

Comment:
This could be expressed with `List.filter`. I these lists are long, using sequences would be better (module `Seq`).